php/uploadPicture.php

```diff
@@ -45,23 +68,37 @@
             ),
             true
         )) {
-        throw new RuntimeException('Invalid file format.');
-    }*/
+        $message = 'Invalid file format.';
+        echo $message;
+        header("Location: ../sites/votingPage.php?project=".$project."&message=".$message);
+        die();
+    }
 
     // You should name it uniquely.
     // DO NOT USE $_FILES['upfile']['name'] WITHOUT ANY VALIDATION !!
     // On this example, obtain safe unique name from its binary data.
+    $format = '../uploads/'.$project.'/%s.%s';
```

Comment:
Ich würde einfach sagen wir hauen später alle bilder in einen Ordner und ordnen die IDs den Projekten zu

Reply:
ne würde ich net machen damit wir es nacher einfacher zipen können

Reply:
Mir fällt grad auf, wir müssen sie Bilder echt irgendwie schützen🤔
Sonst könnte jeder einfach den direkten Link zum Bild kopieren und weiterleiten🤔

Reply:
Ja fände es gut wenn wir sie verschlüsseln, aber jeder kann es eh rechtsklicken und runterladen